src/server/node_manager.c

```diff
@@ -7928,17 +7930,6 @@ set_resv_for_degrade(struct pbsnode *pnode, resc_resv *presv)
 	if (degraded_time > (time_now + resv_retry_time))
 			set_resv_retry(presv, (time_now + resv_retry_time));
 
-	if (presv->ri_resv_retry) {
-		str_time = ctime(&presv->ri_resv_retry);
-		if (str_time != NULL) {
-			str_time[strlen(str_time) - 1] = '\0';
-			(void) snprintf(log_buffer, sizeof(log_buffer),
-				"An attempt to reconfirm reservation will be made on %s",
```

Comment:
Same here, looks like the message has switched with the new behavior

Reply:
Here you are correct.  I think the new behavior is probably right.  Before if a degraded reservation went in-conflict, you'd see 'An attempt' again.  It already got one of those messages when it went degraded in the first time.